test/module/index.js

```diff
@@ -98,44 +98,40 @@ describe('Module Tests', function() {
   it('should throw an error when the first parameter is not defined', function() {
     const lighthouse = require('../..');
     return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is an empty string', function() {
     const lighthouse = require('../..');
-    return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse('')
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is an empty string');
+      }, err => {
+        assert.ok(err);
```

Comment:
I'm using the `.then(onSuccess, onFailure)` form here. The `onSuccess` path will only be called if the call to `lighthouse` succeeded, and then throw an error which will fail the test. The `onFailure` path will only be called if `lighthouse` was rejected, which will be caught in the `onFailure` call and so the test will pass.